libs/ledger-live-common/package.json

```diff
@@ -180,6 +180,7 @@
     "@stacks/transactions": "6.11.0",
     "@stricahq/bip32ed25519": "^1.0.3",
     "@stricahq/typhonjs": "^1.2.6",
+    "@tanstack/react-query": "^5.17.19",
```

Comment:
is there a reason we have different versions in the mono repo, might be better if we match them, this can be helpful in reducing installation and building of the app ... and possibly set it explicitly to the [latest version](https://github.com/TanStack/query/releases/tag/v5.28.7)

Reply:
I'll update them :)

Reply:
More than using the same version I think that we need to declare `@tanstack/react-query` as a `peerDependency` in live-common. Could you give it a try, please?